models/experimental/functional_yolov4/tt/ttnn_downsample3.py

```diff
@@ -0,0 +1,46 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+
+import torch
+import torch.nn as nn
+from models.experimental.functional_yolov4.tt.ttnn_resblock import TtResBlock
+
+from ttnn.model_preprocessing import preprocess_model
+
+import ttnn
+import tt_lib
+
+
+class TtDownSample3:
```

Comment:
please attached the ttnn graph generated my running the module to this PR so we can ensure it matches with the torch graphs.

Reply:
[downsample3_pytoch.gv.pdf](https://github.com/tenstorrent/tt-metal/files/15006509/downsample3_pytoch.gv.pdf)